src/Miso/Html/Element.hs

```diff
@@ -10,119 +10,138 @@
 -- Portability :  non-portable
 ----------------------------------------------------------------------------
 module Miso.Html.Element
-  ( -- *** Smart constructors
+  ( -- ** Smart constructors
       nodeHtml
     , nodeHtmlKeyed
-    -- *** Combinators
-    , doctype_
+    -- ** Document metadata
     , html_
+    , doctype_
+    , base_
     , head_
+    , link_
     , meta_
+    , style
+    , title_
+    -- ** Sectioning root
+    , body_
+    -- ** Content sectioning
+    , address_
+    , article_
+    , aside_
+    , footer_
+    , header_
     , h1_
     , h2_
     , h3_
     , h4_
     , h5_
     , h6_
+    , hgroup_
+    , main_
+    , nav_
+    , section_
+    , search_
+    -- ** Text content
+    , blockquote_
+    , dd_
     , div_
-    , p_
+    , dl_
+    , dt_
+    , figcaption_
+    , figure_
     , hr_
+    , li_
+    , liKeyed_
+    , menu_
+    , ol_
+    , p_
     , pre_
-    , blockquote_
+    , ul_
+    -- ** Inline text semantics
+    , a_
+    , abbr_
+    , b_
+    , bdi_
+    , bdo_
+    , br_
+    , cite_
     , code_
+    , data_
+    , dfn_
     , em_
+    , i_
+    , kbd_
+    , mark_
+    , q_
+    , rp_
+    , rt_
+    , ruby_
+    , s_
+    , samp_
+    , small_
     , span_
-    , a_
     , strong_
-    , i_
-    , b_
-    , u_
     , sub_
     , sup_
-    , br_
-    , ol_
-    , ul_
-    , li_
-    , liKeyed_
-    , dl_
-    , dt_
-    , dd_
+    , time_
+    , u_
+    , var_
+    , wbr_
+    -- ** Image and multimedia
+    , area_
+    , audio_
     , img_
+    , map_
+    , track_
+    , video_
+    -- ** Embedded content
+    , embed_
+    , fencedframe_
     , iframe_
+    , object_
+    , picture_
+    , source_
+    -- ** Scripting
     , canvas_
-    , math_
```

Comment:
Highlight: math_ node should live in MATHML namespace, so I moved this helper to Mathml.Element module and reused it in Mathml example.
I think it would make sense to add all supported mathml elements to that module (https://developer.mozilla.org/en-US/docs/Web/MathML/Reference/Element) but I'll do that in separate PR to keep this PR focused.